### PR TITLE
IL - Add another field to the Commons: Carrying Capacity. This is the number of cows the commons can support

### LINKED ISSUE #27: Add another field to the Commons: Carrying Capacity. This is the number of cows the commons can support.
Each commons should have a "carrying capacity", which is the number of cows the commons can support.

For now, for other issues that depend on this concept, we'll assume a value of 100. But we want that to be adjustable.

In this issue:

- Add carryingCapacity to the Commons entity
- Add it to the CRUD operations for creating Commons
- Add it to the fixtures for commons

### PR DESCRIPTION
**Pull request by ilai-ucsb**
In this PR I added another field called "carrying capacity" to commons. Also updated fixtures but is not yet implemented in storybook.

Creating a new Commons with carrying capacity:
github

closes #27

### CODE REVIEW

**Review by KaieChen — APPROVED** [review]
LGTM!

File: src/test/java/edu/ucsb/cs156/happiercows/controllers/CommonsControllerTests.java [hunk 1]
```diff
@@ -5,6 +5,8 @@
 import java.util.List;
 import java.util.Optional;
 
+import javax.smartcardio.Card;
```
Comment (ethwu) [inline src/test/java/edu/ucsb/cs156/happiercows/controllers/CommonsControllerTests.java:8]: is this necessary for the pr?

**Review by ethwu — COMMENTED** [review]
looks good except for the extra import, I think it's exraneous but I'm not sure @ilai-ucsb ?

File: src/test/java/edu/ucsb/cs156/happiercows/controllers/CommonsControllerTests.java (hunk 1 above)
Comment (ilai-ucsb) [reply src/test/java/edu/ucsb/cs156/happiercows/controllers/CommonsControllerTests.java:8]: Oh yea, it's not necessary. I'll go ahead and remove it.

**Review by RAILGUN1124 — DISMISSED** [review]
lgtm

**Review by blakerandle — APPROVED** [review]
lgtm

**Review by pconrad — DISMISSED** [review]
LGTM... will test on QA

**Review by pconrad — APPROVED** [review]
LGTM.. will test on qa

**Pull request merged**